components/footer/_footer.scss

```diff
@@ -34,3 +34,16 @@
     }
   }
 }
+.footer--eq {
+  .list--bare {
```

Comment:
We shouldn't bring classes in from other `.scss` modules, it makes it difficult to maintain. If we need to do something different if the link is in a list, create a modifier on the link i.e. `.footer__link--list`

Reply:
Should be resolved in the latest commit.